front-end/src/RouterConfig.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import App from "./App";
+import User from "./components/Content/User/User";
+
+export default function RouterConfig() {
+  return (
+    <>
+      <BrowserRouter>
+        <Routes>
+          <Route index element={<App />} />
+          <Route path="/users" element={<User />} />
+        </Routes>
+      </BrowserRouter>
+    </>
+  )
+}
```

Comment:
At the end file we need to add blank line.
The is the rule of Github and ESLint or orther language.